app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: list[str] = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    skills: list[str] = ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        self.skills: list[str] = FrontendDeveloper.skills
```

Comment:
```suggestion
        self.skills: list[str] += ["JavaScript", "HTML", "CSS"]
```

Reply:
It is a better variant, and you must add += to add skills if the class will inherit from two or more classes.

Reply:
change here and all of the code